x/application/keeper/msg_server_stake_application_test.go

```diff
@@ -61,7 +68,11 @@ func TestMsgServer_StakeApplication_SuccessfulCreateAndUpdate(t *testing.T) {
 	require.NoError(t, err)
 	foundApp, isAppFound = k.GetApplication(ctx, appAddr)
 	require.True(t, isAppFound)
-	require.Equal(t, int64(200), foundApp.Stake.Amount.Int64())
+	require.Truef(t,
+		upStake.Amount.Equal(foundApp.Stake.Amount),
+		"expected %d, got %d",
+		upStake.Amount.Int64(), foundApp.Stake.Amount.Int64(),
+	)
```

Comment:
```suggestion
	require.Equal(t, upStake.Amount.Int64(), foundApp.Stake.Amount.Int64())
```

Reply:
I believe you could assert equality even for the coins themselves IIRC

Reply:
I think I got hung up on a pointer and non-pointer not satisfying `Equal()` and just didn't read the error closely enough. :sweat_smile: